CONTRIBUTING_JS.md

```diff
@@ -204,36 +202,6 @@ The header is mandatory and the scope of the header is optional.
 
 Any line of the commit message cannot be longer 100 characters! This allows the message to be easier to read on GitHub as well as in various git tools.
 
-#### Sign-off on commits
-
-We kindly ask and expect for all commits to be signed off with the same license as the repo and module. This can be done by appending the following to your commit message:
-
-```
-License: MIT
-Signed-off-by: User Name <email@address>
-```
-
-where "User Name" is the author's real (legal) name and email@address is one of the author's valid email addresses.
-
-If you want to automatically add this to all of your commits, you can do the following:
-
-Create a file in `~/git-commit-template` with the following contents (notice the two blank lines in the top):
-
-```
-
-
-License: MIT
-Signed-off-by: Victor Bjelkholm <git@victor.earth>
-```
-
-Then, add the following to your global git config (Usually at `~/.gitconfig`):
-
-```
-[commit]
-        template = /home/user/git-commit-template
-```
```

Comment:
This is not required anymore.

Reply:
Interesting, what is the reason?

Reply:
BTW I'm +1 on this - it's a significant hurdle to also ask a new contributor to add a commit sign off message.

Reply:
Just FYI for anyone else reading this, it's not required if we switch to a dual MIT+Apache2 licence since the Apache2 licence states:

> Unless You explicitly state otherwise, any Contribution intentionally submitted for inclusion in the Work by You to the Licensor shall be under the terms and conditions of this License, without any additional terms or conditions.

Reply:
I'm not sure if it should be still part of the guide until we've switched to MIT+Apache.